content/en/security/vulnerabilities/_index.md

```diff
@@ -16,48 +14,58 @@ further_reading:
 
-This opens the CSM [Vulnerabilities][1] page which includes information about: 
+{{< img src="security/vulnerabilities/container_vulnerability_2.png" alt="Container Images overview page highlighting the vulnerabilities tab" width="100%">}}
 
-- A description of the vulnerability.
-- The date on which the vulnerability was last detected.
-- Recommended remediation steps.
-- Affected container images and hosts.
+Within CSM, the severity of a vulnerability is modified from the base score to take into account the business sensitivity of the environment where the vulnerability is detected. For example, if no production environment is detected, the severity is reduced.
 
-{{< img src="security/infrastructure_vulnerabilities/vulnerabilities_side_panel.png" alt="CSM Vulnerabilities side panel showing details of a vulnerability" width="100%">}}
+The Datadog Adjusted Vulnerability Score includes the full context of each impacted container or host:
+
+- The original vulnerability severity
+- Environment the infrastructure is running in
+- Known public exploit
 
 ## Review and remediate
 
-View a consolidated list of vulnerabilities on the [Vulnerabilities][1] page in [Cloud Security Management][3]. Prioritize them for remediation using available facets, such as context-based **severity**, **resource type**, and **library**.
+View a consolidated list of vulnerabilities on the [Vulnerabilities][1] page in [Cloud Security Management][3]. Prioritize them for remediation using available facets, such as context-based **severity**, **resource type**, and **library**.</br>
+
+The vulnerabilities explorer also offers remediation recommendations for detected vulnerabilities that enable you to change the status of a vulnerability, and assign it to a team for further review. It also includes a collection of links and references to websites or information sources that help you understand the context behind each vulnerability.
+
+**Note**: CSM vulnerabilities are auto-closed for infrastructure that is either no longer running, or contains the remediation to the vulnerability.
 
-{{< img src="security/infrastructure_vulnerabilities/vulnerabilities_page.png" alt="A list of high severity vulnerabilities on the CSM Vulnerabilities page" width="100%">}}
+{{< img src="security/vulnerabilities/CSM_vulnerabilities_assign_team.png" alt="CSM vulnerabilities explorer page highlighting the assign a team option" width="100%">}}
 
 [1]: https://app.datadoghq.com/security/infra-vulnerability
 [2]: https://app.datadoghq.com/containers/images
 [3]: https://app.datadoghq.com/security/csm
 [4]: https://app.datadoghq.com/security/infra-vulnerability?query=asset_type%3AHost&group=none
+[5]: /security/application_security/vulnerability_management/
+[6]: https://app.datadoghq.com/security/issues
+[7]: /security/misconfigurations/
+[8]: /security/identity_risks/
```

Comment:
```suggestion
```

Reply:
I'm not seeing these links being reference anywhere, so I think we can go ahead and remove them.